<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>WindowMain</class>
	<widget class="QMainWindow" name="WindowMain">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>290</width>
				<height>201</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Uninstance</string>
		</property>
		<property name="windowIcon">
			<iconset>
				<normaloff>../icons/ToolSeq.svg</normaloff>../icons/ToolSeq.svg
			</iconset>
		</property>
		<widget class="QWidget" name="Widget_00">
			<layout class="QVBoxLayout" name="verticalLayout">
				<property name="spacing">
					<number>2</number>
				</property>
				<property name="leftMargin">
					<number>2</number>
				</property>
				<property name="topMargin">
					<number>2</number>
				</property>
				<property name="rightMargin">
					<number>2</number>
				</property>
				<property name="bottomMargin">
					<number>2</number>
				</property>
				<item>
					<widget class="QWidget" name="Widget_01" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_5">
							<property name="spacing">
								<number>10</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QLabel" name="label_6">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Transforms</string>
									</property>
									<property name="alignment">
										<set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_5">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>:</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="Label_TransformsCount">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>0</string>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_02" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_6">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QPlainTextEdit" name="Text_Transforms">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Expanding">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="lineWrapMode">
										<enum>QPlainTextEdit::NoWrap</enum>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_03" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
							<property name="spacing">
								<number>5</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QCheckBox" name="Check_KeepOld">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Keep Old</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="Button_UninstanceHierarchy">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Uninstance Hierarchy</string>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="Line" name="Widget_04">
						<property name="sizePolicy">
							<sizepolicy hsizetype="Ignored" vsizetype="Fixed">
								<horstretch>0</horstretch>
								<verstretch>0</verstretch>
							</sizepolicy>
						</property>
						<property name="minimumSize">
							<size>
								<width>0</width>
								<height>10</height>
							</size>
						</property>
						<property name="frameShadow">
							<enum>QFrame::Plain</enum>
						</property>
						<property name="lineWidth">
							<number>2</number>
						</property>
						<property name="orientation">
							<enum>Qt::Horizontal</enum>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_05" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
							<property name="spacing">
								<number>5</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QWidget" name="Widget_SourceShape" native="true">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Preferred">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<layout class="QHBoxLayout" name="horizontalLayout_2">
										<property name="spacing">
											<number>5</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QLabel" name="Label_SourceShape">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>Source Shape</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QLabel" name="label_7">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>:</string>
												</property>
												<property name="alignment">
													<set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
							<item>
								<widget class="QLineEdit" name="Line_SourceShape">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_06" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,2">
							<property name="spacing">
								<number>5</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QWidget" name="Widget_TargetShape" native="true">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Preferred">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<layout class="QHBoxLayout" name="horizontalLayout_3">
										<property name="spacing">
											<number>5</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QLabel" name="Label_TargetShape">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>Target Shape</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QLabel" name="label_8">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>:</string>
												</property>
												<property name="alignment">
													<set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
							<item>
								<widget class="QLineEdit" name="Line_TargetShape">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_07" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QPushButton" name="Button_AddInstancedShape">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Add Instanced Shape</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="Button_ReplaceInstancedShape">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Replace Instanced Shape</string>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
			</layout>
		</widget>
	</widget>
	<resources/>
	<connections/>
</ui>